### PR TITLE
[JENKINS-36525] Convert script.js to adjunct

### PR DESCRIPTION
**Pull request by Vlatombe**
Also updated parent POM, missing jelly escapes and removed deprecated
stuff.

https://issues.jenkins-ci.org/browse/JENKINS-36525
@reviewbybees

### CODE REVIEW

**Comment by Vlatombe** [comment]
@jenkinsci/code-reviewers

**Comment by andresrc** [comment]
:bee:

**Comment by recena** [comment]
🐝

**Comment by jglick** [comment]
Works because of [this](https://github.com/jenkinsci/jenkins/blob/f9efe652490976b03b07a669d3e77dba75b8cce7/core/src/main/java/jenkins/model/Jenkins.java#L4773). But a functional test proving it would have been nice.

File: pom.xml [hunk 1]
```diff
@@ -3,17 +3,21 @@
...
-  <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>form-element-path</artifactId>
   <version>1.6-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Form element path plugin</name>
   <description>Introduce unique path attributes for UI controls to facilitate Jenkins UI testing.</description>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/Form+Element+Path+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.609.3</jenkins.version>
```
Comment (olivergondza) [inline pom.xml]: Why so new?

Comment (Vlatombe) [inline pom.xml]: Could be older, but that baseline is already almost a year old..

**Pull request merged**